scripts/build.ps1

```diff
@@ -1,16 +1,19 @@
 ﻿Param(
     [string]$Configuration="Debug",
     [string]$Restore="true",
-    [string]$Channel="preview",
-    [string]$Version="2.0.0-preview1-005675",
+    [string]$Version="<default>",
     [string]$BuildVersion=[System.DateTime]::Now.ToString('eyyMMdd-1')
 )
 
+if ($Version -eq "<default>") {
+    $Version = (Get-Content "$PSScriptRoot\..\DotnetCLIVersion.txt" -Raw).Trim()
```

Comment:
The file has a single line and shouldn't cause any issues with performance or strange newline stuff.